docs/concepts/snapshots.md

```diff
@@ -5,12 +5,67 @@ linkTitle: Ondat Snapshots
+
+A “snapshot” is a point-in-time copy of a PVC. Snapshots are modeled via the
+`VolumeSnapshot` and `VolumeSnapshotContent` Kubernetes API objects. Snapshots
+have limited use as they live within the cluster and cannot be used to restore
+the PVC if the node holding the snapshot is lost.
+
+A “backup” is the process of materialising a new PVC, whose data source is a
+previously created snapshot and then extracting the data to a location outside
+of the cluster. The Ondat Snapshots feature integrates with Kasten K10 to
+provide backup functionality.
+
+A “restore” is the process of restoring an application from a given backup. The
+Ondat Snapshots feature integrates with Kasten K10 to provide restore
+functionality.
 
 # How does it work?
 
-1. The Ondat Snapshots feature takes a snapshot of your volume (i.e. a point-in-time copy of a volume that lives within your cluster). The snapshot now lives within your cluster.
-1. Ondat Snapshots integrates with backup solutions like [Kasten](https://www.kasten.io) to provide backup and restore of Kubernetes stateful applications. Our feature integrates fully with Kasten, which offers free workflows for backing up Kubernetes data (e.g. to an S3 bucket if you are using AWS).
-1. Your volume is now backed up externally and can be retrieved and restored using Kasten.
+The Kubernetes [Volume
+Snapshots](https://kubernetes.io/docs/concepts/storage/volume-snapshots/)
+feature provides users with a set of custom resource definitions (CRD) and APIs
+to create and manage volume snapshots. Storage providers can then implement the
+necessary CSI APIs to integrate with this feature. This is exactly what we’ve
+done at Ondat. Additional tools, like Kasten K10, can then be utilised to
+orchestrate and automate snapshotting, backups and restores. 
+
+Please see the [Backups and restores with Kasten
+K10](/docs/operations/backups-and-restores-with-kastenk10.md) for a full
+walk through.
+
+Note: the Ondat Snapshot feature is not fully CSI compliant. As such the feature
+can only be used with Kasten K10 and with restoration from an external backup.
+
+# Scope and limitations
+
+The feature has the following limitations:
 
-> ⚠️ This feature is currently available on `2.8.0-beta` - if you would like to gain early access to the product, speak with our experts on hello@ondat.io.
+1. The feature has been designed to work with Kasten K10 only. This is not a
+fully CSI compliant implementation of the spec.
+1. Restoring via Kasten 10 from a “local snapshot” is not supported with the
+Ondat Snapshot feature. Users may only restore applications using a Kasten K10
+“External backup”. 
+1. Snapshotting RWX volumes is not supported. This is because it is next to
+impossible to ensure that a NFS mounted volume is in a suitable state for
+snapshotting. For RWX volumes, the user only has access to the filesystem on
+the NFS client. It is not possible to run `fsfreeze` on this mountpoint - NFS
```

Comment:
"It is not possible to run fsfreeze..." -> this should be a separate bullet?

Reply:
Nope it's the same sentence.